test/integration/hasMany.test.js

```diff
@@ -265,6 +265,17 @@ describe('hasMany', function () {
         limit: 1
       }]);
     });
+
+    it('should do stuff', async function() {
+      let project1 = await this.Project.findById(this.project1.id, { include: [ this.Project.associations.members ]});
+      let project2 = await this.Project.findById(this.project2.id, { include: [ this.Project.associations.members ]});
+
+      expect(project1.members, 'not to be undefined');
```

Comment:
Right, project of course ;)

But mick's comment about dataloader is actually more correct, we just want to make sure it completely skips batching